projects/rocblas/CMakeLists.txt

```
@@ -145,17 +145,16 @@ set( GPU_TARGETS "${target_list}" CACHE STRING "List of GPUs (gfx targets) to su
 set( AMDGPU_TARGETS "${GPU_TARGETS}" CACHE STRING "List of GPUs (gfx targets) to support" FORCE ) # for old hip cmake files backward compatibility
 
 option( BUILD_WITH_HIPBLASLT "Build with HipBLASLt" ON )
-set( hipblaslt_path "/opt/rocm" CACHE PATH "Use local HipBLASLt directory" )
 set( HIPBLASLT_VERSION 1.0.0 CACHE STRING "The version of HipBLASLt to be used" )
 
-if( WIN32 OR NOT BUILD_WITH_TENSILE OR NOT BUILD_SHARED_LIBS )
+if( NOT BUILD_WITH_TENSILE )
   set( BUILD_WITH_HIPBLASLT OFF )
 endif()
```

```suggestion
set(BUILD_WITH_HIPBLASLT ${BUILD_WITH_TENSILE})
```
What you have is correct so this is simply a suggestion.

---

I use independent control for both hipblaslt and tensile at times during development.  i.e. Tensile but no hipblaslt.  The original form was just if no Tensile then turn off hipblaslt as well as it requires tensile support layer to work.

---

Got it, in that case we can go with what original is in the patch.